lib/document/templates.js

```diff
@@ -1,12 +1,14 @@
 const fs = require('fs')
 const path = require('path')
+const fileUrl = require('file-url')
 
 const repeatTableHeadersContent = fs.readFileSync(`${__dirname}/repeating-table-headers.js`, 'utf8')
 const pagedContent = fs.readFileSync(require.resolve('pagedjs/dist/paged.polyfill.js'), 'utf8')
 const asciidoctorStyleContent = fs.readFileSync(`${__dirname}/asciidoctor.css`, 'utf8')
 const documentStyleContent = fs.readFileSync(`${__dirname}/document.css`, 'utf8')
 const titleDocumentStyleContent = fs.readFileSync(`${__dirname}/title-document-numbering.css`, 'utf8')
 const titlePageStyleContent = fs.readFileSync(`${__dirname}/title-page-numbering.css`, 'utf8')
+const mathjaxFileUrl = fileUrl(require.resolve("mathjax/unpacked/MathJax.js"))
```

Comment:
I wonder if it's worth it...
In this case I believe that `require.resolve` will always return an absolute path, so we can do the following right?

```suggestion
const mathjaxFileUrl = `file://${require.resolve("mathjax/unpacked/MathJax.js")}`
```

Reply:
There is some extra logic regarding windows file paths; if they start with a drive letter they need to be prefixed. See https://github.com/sindresorhus/file-url/blob/master/index.js -- therefore I chose fileUrl. Otherwise I would need to put the logic in here. I leave the decision to you.

Reply:
> There is some extra logic regarding windows file paths; if they start with a drive letter they need to be prefixed

Windows paths... *sigh*
Having said, I don't think it applies here because browsers (at least Firefox and Chrome) are smart enough to resolve it. The following is working on Windows:

```html
<script src='file://c:/path/to/hello.js'></script>
```

In the network tab, I can see that the requested URL is in fact `file:///C:/dev/hello.js`.
But `file-url` is also encoding the URI so it's not as simple as adding `file://` as a prefix.

> Otherwise I would need to put the logic in here. I leave the decision to you.

Tough decision... but since this dependency is small, has tests and no dependency, let's keep it that way 😅